GliaWidgets/Layout/Makeable.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+
+protocol Makeable {}
+
+extension Makeable where Self: UIView {
```

Comment:
Having specific DSL for creating views while being extremely helpful, it makes support harder for those who are not familiar with this DSL, especially when it's not documented/tested. Is to possible omit usage of Makable? Would it make declaration of UIView more verbose?

Reply:
I've added docs here - 43d3ed6b307908ef0f23ec2e657961394f864243
I just wanted to highlight that this approach is more about creation and setting up objects in one place and avoiding spearing initialization and setting up in different places of application/file.